wiki/Rules/Content_Usage_Guidelines/en.md

```diff
@@ -0,0 +1,39 @@
+# Content Usage Guidelines
+
+A user should have permission from relevant copyright holders for audio/visual/gameplay assets included in their beatmap submissions. Without permission, the beatmap is at risk of being removed and the creator is at risk of being denied access to the submission system as per the [osu! copyright policy](/wiki/legal/copyright).
```

Comment:
```suggestion
As a mapper, you should have permission from relevant copyright holders for audio/visual/gameplay assets included in their beatmap submissions. Without permission, the beatmap is at risk of being removed and the creator is at risk of being denied access to the submission system as per the [osu! copyright policy](/wiki/legal/copyright).
```

I think we should address as if a mapper is coming to this page, to make it more personal.